src/main/java/com/oath/halodb/HaloDB.java

```diff
@@ -88,6 +88,10 @@ public void pauseCompaction() throws HaloDBException {
         }
     }
 
+    public void snapshot() {
```

Comment:
Need another API to delete snapshot

Reply:
I'm not sure if do we really need the delete function. I'll try to add a new function getSnapshotPath to let the client to make the decision.